### PR TITLE
The target application id field is a dropdown

### LINKED ISSUE #44: Make the target applications choice a dropdown
This is a request from @jvillalobos

I believe the available target applications are (from https://github.com/mozilla/addons-server/blob/master/src/olympia/constants/applications.py):
- Firefox: {ec8030f7-c20a-464f-9b0e-13a3a9e97384}
- Thunderbird: {3550f703-e582-4d05-9a08-453d09bdfdc6}
- Seamonkey: {92650c4d-4b8e-4d2a-b7eb-24ecf4f6b63a}
- Sunbird: {718e30fb-e89b-41dd-9da7-e25a45638b28}
- Mobile: {a23983c0-fd0e-11dc-95ff-0800200c9a66}
- Android: {aa3c5121-dab2-40e2-81ca-7ea25febc110}
- Mozilla: {86c18b42-e466-45a9-ae7a-9b95ba6f5640}

Not sure all of those are actually used in the xml file.

### PR DESCRIPTION
**Pull request by magopian**
Fixes #44

@natim r?

### CODE REVIEW

File: amo-blocklist.json [hunk 1]
```diff
@@ -120,7 +120,13 @@
                     "id": {
                       "type": "string",
                       "title": "Application id",
-                      "description": "The application unique identifier."
+                      "description": "The application unique identifier.",
+                      "enum": [
+                        "{ec8030f7-c20a-464f-9b0e-13a3a9e97384}",
```
Comment (Natim) [inline amo-blocklist.json]: How do you plan on settings this with a readable information on the kinto-admin side?

Comment (magopian) [inline amo-blocklist.json]: It needs `enumNames` (or `oneOf`) in `react-jsonschema-form`, as explained [here](https://github.com/mozilla-services/amo-blocklist-ui/issues/44#issuecomment-194881471). I filed #54 to add meaningful labels once that's implemented.

**Comment by magopian** [comment]
rebased

**Comment by magopian** [comment]
Actually, this might be an issue: https://github.com/mozilla-services/react-jsonschema-form/issues/37

**Comment by Natim** [comment]
Is there any way of having the enum label somewhere?

**Comment by n1k0** [comment]
Not until we support `enumNames` in react-jsonschema-form, see https://github.com/mozilla-services/react-jsonschema-form/issues/25

**Comment by magopian** [comment]
Filed #44 for the usage of `enumNames`

**Pull request merged**